### PR TITLE
Character Setup Traits

### LINKED ISSUE #6109: trait selection flavor text doesnt update very quickly
Submitter: tunnelflip
I dont know if its a byond thing, but moving the window seems to update it, there is a text so you can read about the trait when you hover your mouse over it but it takes to long for it to show.

### PR DESCRIPTION
**Pull request by Luxizzle**
## About the PR 

Moves traits into the character setup window.

Todo:
- [x] Display trait icons
  
  Currently we have no nice way of displaying icons in tgui windows. We should either fix that or we can move the trait icons from the dmi file into the `browserassets` folder as pngs, since I'm pretty sure they are not used anywhere else anyways.

yharSbaWjs

## Why's this needed? 

Janky old window needs to be removed.

Fixes #6109

## Changelog

```changelog
(u)Luxizzle
(*)Moved traits into the character setup window.
```

### CODE REVIEW

**Comment by stylemistake** [comment]
> Not same width

Add `basis={0}` alongside `grow`.

**Comment by Luxizzle** [comment]
I'm crying.

**Comment by Gannets** [comment]
Can't speak to code quality but this is very much needed. Really like consolidation of this stuff in to one window.

**Comment by mordent-goonstation** [comment]
Will look at when I have good internet next week, please hold merging until then.

File: code/datums/preferences.dm [hunk 1]
```diff
@@ -146,6 +146,20 @@ datum/preferences
 		user << browse_rsc(icon(cursors_selection[target_cursor]), "tcursor_[src.target_cursor].png")
 		user << browse_rsc(icon(hud_style_selection[hud_style], "preview"), "hud_preview_[src.hud_style].png")
 
+		var/list/traits = list()
+		for (var/obj/trait/C in src.traitPreferences.getTraits(user))
```
Comment (ZeWaka) [inline code/datums/preferences.dm:150]: `as anything` applicable here? i think so.

**Comment by Luxizzle** [comment]
Also uh, please actually fix cdn before merging this or people will cry over not being able to modify traits

**Pull request closed without merging**